include/debug/tracing.h

```diff
@@ -1,139 +1,12 @@
-/*
- * Copyright (c) 2018 Intel Corporation
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-#ifndef ZEPHYR_INCLUDE_DEBUG_TRACING_H_
-#define ZEPHYR_INCLUDE_DEBUG_TRACING_H_
+ /*
+  * Copyright (c) 2020 Intel Corporation
+  *
+  * SPDX-License-Identifier: Apache-2.0
+  */
 
-#include <kernel.h>
+#ifndef ZEPHYR_INCLUDE_DEBUG_TRACING_H
```

Comment:
Maybe `ZEPHYR_INCLUDE_DEBUG_TRACING_H` to `ZEPHYR_INCLUDE_TRACING_H`? It's not critical.